stdlib/InteractiveUtils/src/editless.jl

```diff
@@ -54,7 +54,12 @@ function edit(path::AbstractString, line::Integer=0)
         cmd = line != 0 ? `$command $path -l $line` : `$command $path`
     elseif startswith(name, "subl") || startswith(name, "atom")
         cmd = line != 0 ? `$command $path:$line` : `$command $path`
-    elseif name == "code" || (Sys.iswindows() && uppercase(name) == "CODE.EXE")
```

Comment:
BTW I'm not sure why L57 is simply not doing startswith(name, "code") , like the rest of the function.

Reply:
https://github.com/JuliaLang/julia/pull/21106#discussion_r107256282

Reply:
thanks